src/Command.php

```diff
@@ -162,7 +162,7 @@ public function getExecCommand()
     {
         if ($this->_execCommand===null) {
             $command = $this->getCommand();
-            if (!$command) {
+            if (!$command || !is_executable($command)) {
```

Comment:
@cyrrill Sorry, this doesn't work either. `is_executable()` expects a full path. But the class also works fine with commands like `ls -l` (see the tests). I also don't really like the approach with `whereis` because this makes other assumptions which may not hold true on some systems (`whereis` present and executable by PHP). So I think there's not really a good solution for this problem.